engine-tests/src/tests/uniswap.rs

```diff
@@ -28,7 +28,7 @@ fn test_uniswap_input_multihop() {
     let mut context = UniswapTestContext::new("uniswap");
 
     // evm_gas = 970k
-    // near total gas = 163 Tgas
+    // near total gas = 122 Tgas
```

Comment:
👀 Is this expected? Is uniswap using alt_bn128?

Reply:
It is actually 122, it just wasn't updated.